<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vote</class>
 <widget class="QWidget" name="vote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stack">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>750</width>
       <height>600</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="vote_page">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>304</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>偏爱的影片选择：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>429</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>偏爱的观影日期：</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>450</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="Vote">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>520</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>进行投票</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>62</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>电影排片表：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_movie">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>320</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>181</x>
         <y>379</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 290 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>偏爱的时段选择：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_time">
       <property name="geometry">
        <rect>
         <x>371</x>
         <y>384</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>631</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>284</x>
         <y>4</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="assign_search">
       <property name="geometry">
        <rect>
         <x>432</x>
         <y>0</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>查看当日排片表</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="none_2"/>
     <widget class="QWidget" name="none"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
